stripe/src/test/java/com/stripe/android/paymentsheet/PaymentSheetAddCardFragmentTest.kt

```diff
@@ -330,6 +332,104 @@ class PaymentSheetAddCardFragmentTest {
         }
     }
 
+    @Test
+    fun `when postal code is valid then billing error is invisible`() {
+        createFragment { _, viewBinding ->
+            assertThat(viewBinding.cardErrors.isVisible)
+                .isFalse()
+
+            viewBinding.billingAddress.countryView.setText("United States")
+            viewBinding.billingAddress.postalCodeView.setText("94107")
+
+            assertThat(viewBinding.cardErrors.text.toString())
+                .isEmpty()
+            assertThat(viewBinding.cardErrors.isVisible)
+                .isFalse()
```

Comment:
Should this be billingErrors instead of cardErrors?

Reply:
yes, thanks for spotting this